app/src/pages/inside/stepPage/modals/linkIssueModal/linkIssueModal.jsx

```diff
@@ -0,0 +1,263 @@
+      intl,
+      url,
+      data: { items, fetchFunc },
+    } = this.props;
+    const testItemIds = items.map((item) => item.id);
+    const issues = formData.issues.map((issue) => ({
+      ticketId: issue.issueId,
+      url: issue.issueLink,
+    }));
+
+    fetch(url, {
+      method: 'put',
+      data: {
+        issues,
+        systemId: formData.systemId,
+        testItemIds,
+      },
+    }).then(() => {
+      fetchFunc();
+      this.props.showNotification({
+        message: intl.formatMessage(messages.linkIssueSuccess),
+        type: NOTIFICATION_TYPES.SUCCESS,
+      });
+    });
+    this.closeModal();
+  };
+
+  handleIssueLinkChange = (event, newValue, previousValue, name) => {
+    const issueIndex = /\d+/.exec(name)[0];
+    let issueIdAutoValue = newValue.split('/');
+    issueIdAutoValue = issueIdAutoValue[issueIdAutoValue.length - 1];
+    this.props.change(`issues[${issueIndex}].issueId`, issueIdAutoValue);
+    this.handleInputChange();
+  };
+
+  handleInputChange = () =>
+    this.setState({
+      isInputsHaveBeenChanged: true,
+    });
+
+  isMultipleExternalSystems = () => this.props.externalSystems.length > 1;
+
+  renderIssues = ({ fields }) => (
+    <ul>
+      {fields.map((issue, index) => (
+        <li key={issue} className={cx('issue-fields-list-item')}>
+          {fields.length > 1 && (
+            <div className={cx('remove-issue-fields-button')} onClick={() => fields.remove(index)}>
+              {Parser(CloseIcon)}
+            </div>
+          )}
+          <FormField
+            name={`${issue}.issueLink`}
+            containerClassName={cx('inputs-group-block')}
+            fieldWrapperClassName={cx('dropdown-container')}
+            label={this.props.intl.formatMessage(messages.issueLinkLabel)}
+            onChange={this.handleIssueLinkChange}
+            labelClassName={cx('multiple-systems-title')}
+            descriptionClassName={cx('none-description')}
+          >
+            <FieldErrorHint>
+              <Input placeholder={this.props.intl.formatMessage(messages.issueLinkPlaceholder)} />
+            </FieldErrorHint>
+          </FormField>
+          <FormField
+            name={`${issue}.issueId`}
+            containerClassName={cx('inputs-group-block')}
+            fieldWrapperClassName={cx('dropdown-container')}
+            label={this.props.intl.formatMessage(messages.issueIdLabel)}
+            onChange={this.handleInputChange}
+            labelClassName={cx('multiple-systems-title')}
+            descriptionClassName={cx('none-description')}
+          >
+            <FieldErrorHint>
+              <Input placeholder={this.props.intl.formatMessage(messages.issueIdLabel)} />
+            </FieldErrorHint>
+          </FormField>
+        </li>
+      ))}
+      <li className={cx('add-issue-button')}>
+        <GhostButton
+          type="button"
+          onClick={() => fields.push({})}
+          icon="+"
+          iconClass={cx('add-issue-button-icon')}
```

Comment:
Use SVG icon for a '+'. And iconClass property becomes unnecessary.

Reply:
Done